test/resources.go

```diff
@@ -65,13 +65,36 @@ func NewTestScheme() *runtime.Scheme {
 }
 
 func NewContainerJFR() *rhjmcv1beta1.ContainerJFR {
+	return &rhjmcv1beta1.ContainerJFR{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "containerjfr",
+			Namespace: "default",
+		},
+		Spec: rhjmcv1beta1.ContainerJFRSpec{
+			Minimal:            false,
+			TrustedCertSecrets: []rhjmcv1beta1.CertificateSecret{},
+		},
+	}
+}
+
+func NewContainerJFRWithSecrets() *rhjmcv1beta1.ContainerJFR {
+	key := "tls.crt"
```

Comment:
Let's use a different key name here. Just to differentiate from the default key. It'll make for a more robust test.